manual/contents/src/operations/configuration-reference.asciidoc

```diff
@@ -22,10 +19,9 @@ include::{importdir}/neo4j-config-docs-docs-jar/ops/configuration-bolt-connector
 
 == Configuring HTTP Connectors
 
-HTTP Connectors expose the Neo4j REST API and related HTTP resources.
-At this time, you can configure at most one HTTP connector and one HTTPS connector.
-
+HTTP Connectors expose the Neo4j's HTTP endpoints.
 HTTPS connectors are configured by setting a connector to require encryption.
+There must be exactly one HTTP connector and zero or one HTTPS connectors configured.
```

Comment:
My gut says otherwise -- but my brain is unable to rationalize it. I suggest we go with my gut on this occasion on account of its geographical location.